src/chain_sync/chain_muxer.rs

```diff
@@ -1,933 +1,23 @@
 // Copyright 2019-2025 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
```

Comment:
Should this file be deleted?

Reply:
The file contains the `SyncConfig`, which is still needed. Would be nice to refactor but that can come in a later PR.